death-forecast/epiforecasts-ensemble1/2020-06-12-epiforecasts-ensemble1.csv

```diff
@@ -0,0 +1,97 @@
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.15,6125.700000000001
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.2,6164
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.25,6197.25
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.3,6231.7
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.35,6261.299999999999
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.4,6289
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.45,6314
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.5,6343.5
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.55,6369
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.6,6398
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.65,6422.700000000001
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.7,6451
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.75,6482.25
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.8,6521
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.85,6564.15
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.9,6627.1
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.95,6690.199999999999
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.975,6786.025
+2020-06-12,1 wk ahead inc death,2020-06-13,US,quantile,0.99,6825.01
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.01,3688.99
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.025,3787.975
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.05,3860.9500000000003
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.1,3970.8999999999996
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.15,4058.7000000000003
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.2,4110.8
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.25,4159.75
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.3,4211
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.35,4255
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.4,4291.6
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.45,4329.55
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.5,4363
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.55,4400.45
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.6,4438.4
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.65,4491.35
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.7,4533.6
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.75,4581.25
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.8,4634
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.85,4698
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.9,4789.3
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.95,4889.4
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.975,4973.025
+2020-06-12,2 wk ahead inc death,2020-06-20,US,quantile,0.99,5089.659999999999
+2020-06-12,1 wk ahead inc death,2020-06-13,US,point,NA,6343.5
```

Comment:
i see NA for the quantule (for the point forecast) I assume this is what they want but just flagging